charts/kubernetes-external-secrets/templates/deployment.yaml

```diff
@@ -40,6 +40,10 @@ spec:
             value: {{ $value | quote }}
           {{- end }}
           {{- end }}
+          {{- if .Values.scopeNamespace.enabled }}
+          - name: "SCOPE_NAMESPACE"
+            value: {{ .Release.Namespace | quote }}
```

Comment:
I couldn't think of a use case for setting the namespace to anything other than `{{ .Release.Namespace }}`

Reply:
think this is fair, if someone comes up with a use case it should be easy enough to change in a future release!